### ISSUE TITLE
Fix broken Travis CI build

### ISSUE THREAD

**Issue by mpenkov** [RaRe-Technologies/gensim#3092]
Continued from #3083. The Travis build (for aarm64) is [failing](https://travis-ci.com/github/RaRe-Technologies/gensim/builds/221428340) for multiple reasons:

- Python 3.6: passes
- Python 3.7: failed unit test: TestTranslationMatrix.test_persistence, numpy raises ValueError
- Python 3.8: failed unit test, as above
- Python 3.9: missing BLAS/LAPACK

@janaknat Can you please investigate?

**Comment by janaknat** [comment]
@mpenkov python 3.9 wheels for numpy and scipy were released in v1.19.3 and v1.5.4. With that change I am seeing the same 5 unit tests failed as seen with python 3.7 and 3.8.

Build logs python 3.9: https://travis-ci.com/github/janaknat/gensim/jobs/494600398

**Comment by mpenkov** [comment]
OK, so if we fix the numpy/scipy versions for python 3.9, we'll have the same result as 3.7 and 3.8.

How do we fix 3.7 and 3.8 builds?

**Comment by janaknat** [comment]
Looks like numpy is having issues. I don't remember seeing this when I made the PR.

**Comment by mpenkov** [comment]
Yes, numpy is having multiple issues. Are you able to investigate?

**Comment by janaknat** [comment]
I can create a patch with oldest-supported-numpy and see if that helps.

**Comment by mpenkov** [comment]
Sure, give it a shot.
